oh fun!

```
Undefined symbols for architecture x86_64:
  "_quadmath_snprintf", referenced from:
      _write_complex in libgfortran.a(write.o)
      _write_float_0 in libgfortran.a(write.o)
      __gfortrani_write_real in libgfortran.a(write.o)
      __gfortrani_write_real_g0 in libgfortran.a(write.o)
  "_strtoflt128", referenced from:
      __gfortrani_convert_real in libgfortran.a(read.o)
      __gfortrani_convert_infnan in libgfortran.a(read.o)
ld: symbol(s) not found for architecture x86_64
collect2: error: ld returned 1 exit status
```

--- a/recipe/build.sh
+++ b/recipe/build.sh
@@ -137,7 +137,7 @@ if [[ "$host_platform" == "$target_platform" ]]; then
   rm ${PREFIX}/lib/libgfortran.spec
   sed "s#@CONDA_PREFIX@#$PREFIX#g" $RECIPE_DIR/libgfortran.spec > ${PREFIX}/lib/libgfortran.spec
 
-  for file in libgfortran.spec libgomp.spec libgfortran.a libgomp.a; do
+  for file in libgfortran.spec libgomp.spec; do
     mv $PREFIX/lib/$file $PREFIX/lib/gcc/${TARGET}/${gfortran_version}/$file
     ln -s $PREFIX/lib/gcc/${TARGET}/${gfortran_version}/$file $PREFIX/lib/$file
   done

--- a/recipe/gfortran_test.sh
+++ b/recipe/gfortran_test.sh
@@ -5,11 +5,11 @@ set -xe
 ${PREFIX}/bin/${macos_machine}-gfortran --help
 
 if [[ "$target_platform" == "$cross_target_platform" ]]; then
-  "${PREFIX}/bin/${macos_machine}-gfortran" -o hello hello.f90
+  "${PREFIX}/bin/${macos_machine}-gfortran" -o hello hello.f90 -v
   ./hello
   rm -f hello
 
-  "${PREFIX}/bin/${macos_machine}-gfortran" -O3 -fopenmp -ffast-math -o maths maths.f90
+  "${PREFIX}/bin/${macos_machine}-gfortran" -O3 -fopenmp -ffast-math -o maths maths.f90 -v
   ./maths
   rm -f maths
 fi

--- a/recipe/libgfortran.spec
+++ b/recipe/libgfortran.spec
@@ -5,7 +5,7 @@
 #
 
 %rename lib liborig
-*lib:  -lm %(libgcc) %(liborig) -rpath @CONDA_PREFIX@/lib
+*lib:  -lquadmath -lm %(libgcc) %(liborig) -rpath @CONDA_PREFIX@/lib
 
 *libgcc:
 %{static-libgcc|static:                                                          %{!m32:%:version-compare(>= 10.6 mmacosx-version-min= -lSystem)}                  -lgcc_eh -lgcc;                                                             shared-libgcc|fexceptions|fgnu-runtime:                                          %:version-compare(!> 10.5 mmacosx-version-min= -lSystem)                  %:version-compare(>< 10.5 10.6 mmacosx-version-min= -lSystem)          %:version-compare(!> 10.5 mmacosx-version-min= -lSystem)             %:version-compare(>= 10.5 mmacosx-version-min= -lSystem)                  -lgcc ;                                     :%:version-compare(>< 10.3.9 10.5 mmacosx-version-min= -lSystem)        %:version-compare(>< 10.5 10.6 mmacosx-version-min= -lSystem)          %:version-compare(!> 10.5 mmacosx-version-min= -lSystem)           %:version-compare(>= 10.5 mmacosx-version-min= -lSystem)                  -lgcc }

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -90,8 +90,8 @@ outputs:
       # fortran module files
       - lib/gcc/{{ chost }}/{{ gfortran_version }}/finclude/**
       # Static libraries and object files
-      - lib/gcc/{{ chost }}/{{ gfortran_version }}/libgomp.a
-      - lib/gcc/{{ chost }}/{{ gfortran_version }}/libgfortran.a
+      #- lib/gcc/{{ chost }}/{{ gfortran_version }}/libgomp.a
+      #- lib/gcc/{{ chost }}/{{ gfortran_version }}/libgfortran.a
       - lib/gcc/{{ chost }}/{{ gfortran_version }}/libgcc.a
       - lib/gcc/{{ chost }}/{{ gfortran_version }}/libgcc_eh.a
       - lib/gcc/{{ chost }}/{{ gfortran_version }}/crtfastmath.o

